pallets/sp-mvm/src/balance.rs

```diff
@@ -53,10 +53,13 @@ where
             address,
             ticker
         );
-        let address = address_to_account::<T::AccountId>(&address).unwrap();
-        <balances::Module<T> as Currency<T::AccountId>>::free_balance(&address)
-            .try_into()
-            .map_err(|_err| error!("Convert native balance to VM balance type."))
+        address_to_account::<T::AccountId>(&address)
+            .map_err(|_| error!("Can't convert address from Move to Substrate."))
+            .and_then(|address| {
+                <balances::Module<T> as Currency<T::AccountId>>::free_balance(&address)
+                    .try_into()
+                    .map_err(|_err| error!("Convert native balance to VM balance type."))
```

Comment:
Is this error really possible?

Reply:
Dunno, but if understood @fzzr- correctly, reviewers hate unwraps

Reply:
imo in case of impossible errors we just need to replace `unwrap` with `expect` with readable error message.

Reply:
It's really possible because there's address conversion over abstract (I mean "generic") type, also same for numeric conversion from/to `u128`.

Reply:
In case when we configuring own runtime it's can be impossible. But not for "just pallet" usage for any other runtime it sims to be possible.